pkg/utils/helm/helm.go

```diff
@@ -93,7 +93,10 @@ func GenerateHelmTemplate(name string, namespace string, valueFileName string, c
 	// return syscall.Exec(cmd, args, env)
 	// 5. execute the command
 	log.V(1).Info("Generating template", "args", args)
-	cmd := exec.Command("bash", "-c", strings.Join(args, " "))
+	cmd, err := utils.SafeCommand("bash", "-c", strings.Join(args, " "))
```

Comment:
This seems in a unused function? How about removing the function?

Reply:
done